build.gradle

```diff
@@ -11,6 +11,7 @@ repositories {
         url 'https://github.com/'
         patternLayout {
             artifact '/[organization]/[module]/releases/download/[revision]/[ext]'
+            artifact '/[organization]/[module]/raw/[revision]/jars/[ext]'
```

Comment:
This is just in case we ever needed to pull a jar directly from a repo. The line above it is for jars posted in release, direct or zipped (see zipped configuration, used for LunaLib and ConsoleCommands deps).